[2023-03-01 12:58:19,011] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_age 2022-11-25T03:05:00+00:00 [queued]>
[2023-03-01 12:58:19,103] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_age 2022-11-25T03:05:00+00:00 [queued]>
[2023-03-01 12:58:19,106] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 12:58:19,116] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2023-03-01 12:58:19,117] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 12:58:19,385] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): get_age> on 2022-11-25T03:05:00+00:00
[2023-03-01 12:58:19,450] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'with_python_operator', 'get_age', '2022-11-25T03:05:00+00:00', '--job-id', '3922', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/02_python_operator.py', '--cfg-path', '/tmp/tmp7si_5cw3', '--error-file', '/tmp/tmpgxa5usw8']
[2023-03-01 12:58:19,453] {standard_task_runner.py:77} INFO - Job 3922: Subtask get_age
[2023-03-01 12:58:19,420] {standard_task_runner.py:52} INFO - Started process 16523 to run task
[2023-03-01 12:58:19,787] {logging_mixin.py:104} INFO - Running <TaskInstance: with_python_operator.get_age 2022-11-25T03:05:00+00:00 [running]> on host abe4c4997ef7
[2023-03-01 12:58:20,188] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dataheroes
AIRFLOW_CTX_DAG_ID=with_python_operator
AIRFLOW_CTX_TASK_ID=get_age
AIRFLOW_CTX_EXECUTION_DATE=2022-11-25T03:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-25T03:05:00+00:00
[2023-03-01 12:58:20,379] {python.py:118} INFO - Done. Returned value was: None
[2023-03-01 12:58:20,555] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=with_python_operator, task_id=get_age, execution_date=20221125T030500, start_date=20230301T125819, end_date=20230301T125820
[2023-03-01 12:58:21,661] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-03-01 12:58:21,751] {local_task_job.py:146} INFO - Task exited with return code 0
